db/seeds/routes_from_varnish.rb

```diff
@@ -5,42 +5,42 @@
 #
 ######################################
 
-unless ENV['GOVUK_APP_DOMAIN'].present?
-  abort "GOVUK_APP_DOMAIN is not set.  Maybe you need to run under govuk_setenv..."
-end
-
-backends = {
-  'canary-frontend' => {'tls' => false},
-  'licensify' => {'tls' => true},
-  'tariff' => {'tls' => false},
-}
-
-backends.each do |name, properties|
-  if properties['tls'] == true
-    protocol = 'https'
-  else
-    protocol = 'http'
-  end
-  url = "#{protocol}://#{name}.#{ENV['GOVUK_APP_DOMAIN']}/"
-  puts "Backend #{name} => #{url}"
-  be = Backend.find_or_initialize_by(:backend_id => name)
-  be.backend_url = url
-  be.save!
-end
+require "gds_api/publishing_api/special_route_publisher"
+require "securerandom"
 
 routes = [
-  %w(/apply-for-a-licence prefix licensify),
-
-  %w(/trade-tariff prefix tariff),
-
-  %w(/__canary__ exact canary-frontend),
+  {
+    content_id: "a652fcab-7aa6-42e5-9d72-c82d7e3c5377",
+    base_path: "/apply-for-a-licence",
+    type: "prefix",
+    rendering_app: "licensify",
+    title: "Apply for a licence",
+    description: "Redirects to the licence finder."
+  },
```

Comment:
Doh, missed that detail.  I wonder if it would be clearer to just include it up here. It's only 2 entries, so the duplication won't be that much of a problem.